userProfile/userProfile.mjs

```diff
@@ -0,0 +1,68 @@
+
+import DatabaseDriver from "../database/driver.mjs"
+
+// This module will use the TinyPEN API (RERUM Mongo DB)
```

Comment:
You are using the private TPEN Mongo Database (`users` collection) not the TinyPEN API

Reply:
As of now i update it, But for this sprint i dont need a DB driver so i had commented it out.

I was facing exception using const database = new DatabaseDriver("mongo") 

passing "mongo" as parameter , if im doing it wrong please let me know what to use in here.

But this is not effective for the current sprint , So as i request for a pulll request approval.